File: loianeTraining/javaBasico/exercicios-aulas-18-19/Ex26.java
public class Ex26 {
    public static void main(String[] args) {
        int[] a = { 1, 2, 7, 4, 5, 6, 7, 8, 9, 10 };
        int[] b = { 10, 9, 7, 7, 6, 5, 3, 3, 2, 1 };
        int[] c = new int[10];

        for (int i = 0; i < c.length; i++) {
            if (a[i] > b[i]) {
                c[i] = 1;
            } else if (a[i] == b[i]) {
                c[i] = 0;
            } else {
                c[i] = -1;
            }
            System.out.println(c[i]);
        }
    }
}

File: loiane-training-csharp/procedural-poo/d/Ex26.cs
using System;

class Ex26
{
    static void Main(string[] args)
    {
        int votosGreg = 0;
        int votosChris = 0;
        int votosCaruso = 0;

        Console.WriteLine("Digite quantas pessoas tem para votar");
        int votos = int.Parse(Console.ReadLine());

        Console.WriteLine("Opcoes de votos: [1] - Greg | [2] - Chris | [3] - Caruso");
        for (int i = 0; i < votos; i++)
        {
            Console.Write("Digite para qual candidato o voto sera destinado: ");
            int voto = int.Parse(Console.ReadLine());

            if (voto == 1)
            {
                votosGreg++;
            }
            else if (voto == 2)
            {
                votosChris++;
            }
            else
            {
                votosCaruso++;

            }
        }
        Console.WriteLine($"Greg tem {votosGreg} votos, Chris tem {votosChris} votos, Caruso tem {votosCaruso} votos.");
    }
}